src/pricing/error.cairo

```diff
@@ -13,5 +13,11 @@ mod PricingError {
         data.append(short_open_interest.into());
         panic(data)
     }
-    const USD_DELTA_EXCEEDS_POOL_VALUE: felt252 = 'usd_delta_exceeds_pool_value';
+
+    fn USD_DELTA_EXCEEDS_POOL_VALUE(usd_delta: felt252, pool_usd_for_token: u128) {
+        let mut data = array!['usd_delta_exceeds_pool_value'];
+        // data.append(usd_delta.into());
```

Comment:
Why comment that?

Reply:
It's crashing, couldn't find why

Reply:
If you think we are gonna reuse it later add a TODO crashing so we can fix it later and not forget